src/wasm-lib/kcl/tests/artifact_graph_sketch_on_face_etc/artifact_graph_mind_map.snap.md

```diff
@@ -221,15 +41,12 @@ mindmap
           Wall
           SweepEdge Opposite
           SweepEdge Adjacent
-        Segment
-          Wall
-          SweepEdge Opposite
-          SweepEdge Adjacent
```

Comment:
I'm pretty sure these lines are deleted because this block was merged with the one above, and this segment is already defined on lines 6-9.